.gitignore

```diff
@@ -4,4 +4,6 @@ node_modules
 _site
 *~
 .DS_Store
-*.off
\ No newline at end of file
+*.off
+
+dist/plugins-2023/plugins_*.html
```

Comment:
this should not be necessary - just don't add these files to the main branch

Reply:
Grand, so use the data to generate the pages in dist and then not commit to github - no worries

Reply:
if they are generated files, then they should be committed
i though they might be manual backup files, but if not then they are integral to the source
in general we do want to commit everything in dist (this is not C, these are not binary object files)

Reply:
No worries, just part of the process as they're ejs files originally